pkg/pillar/dpcmanager/dpcmanager.go

```diff
@@ -33,6 +33,20 @@ const LastResortKey = "lastresort"
 
 var nilUUID = uuid.UUID{} // used as a constant
 
+// DevicePortConfig is an interface to manage a list of received device port configurations
+// see also: DpcManager
+type DevicePortConfig interface {
```

Comment:
`DevicePortConfig` refers to data type: https://github.com/lf-edge/eve/blob/master/pkg/pillar/types/zedroutertypes.go#L477
Interface name should refer to an actor, commonly ending with `er`, e.g. Manager, Stringer, etc.
But I think that mocking DPCManager just to pass DNS to the `resolve...` function in UTs just unnecessary adds too much new and mostly dead code, see my comment for `resolveWithPortsLambda`.

Reply:
As the `resolveWith...` methods were made independent from the NIM struct, I was able to remove the `DevicePortConfig` completely. That made the code change a lot smaller - thx for the suggestion.